### PR TITLE
Fix Bug 1115285 - [l10n] Thunderbird start page: make support link localizable

### PR DESCRIPTION
**Pull request by kyoshino**

### CODE REVIEW

File: bedrock/thunderbird/templates/thunderbird/start/release.html [hunk 1]
```diff
@@ -31,8 +31,9 @@
       <div>
         <section class="help">
           <h2 class="iconic">{{ _('Need Support?') }}</h2>
+          {% set support_url = '/questions/new/thunderbird' if LANG in ['en-GB', 'en-US', 'en-ZA'] else '/kb/get-community-support' -%}
```
Comment (alexgibson) [inline bedrock/thunderbird/templates/thunderbird/start/release.html]: Would it be better to check for locales that start with `en-` here as opposed to enumerate through the hard-coded list?

**Comment by kyoshino** [comment]
Looks like the forum link can be localizable freely. I have just updated the PR.

**Comment by alexgibson** [comment]
This does create the possibility for localizers to break the page if the link is formatted incorrectly - thoughts @flodolo?

**Comment by flodolo** [comment]
I was one who proposed this solution in the first place ;-)
We are already using the same strategy on the first run page for Nightly, and most locales will keep the English string. 

Anyhow, all .lang files are manually checked (and pages tested) before going out to production and being enabled, so that's a risk that I'm willing to take.

**Comment by alexgibson** [comment]
Thanks for the clarification @flodolo :+1:

**Comment by alexgibson** [comment]
And thank you @kyoshino!

**Pull request merged**